_config.yml

```diff
@@ -402,3 +402,18 @@ contributors:
       twitter: https://twitter.com/isayjilh
       github: https://github.com/jilh
       linkedin: https://www.linkedin.com/in/jilh
+"Deji Adesoga":

```

Comment:
The YAML format is using tabs to create **a list item**. This is not valid. Please look at the examples above.